kolibri/plugins/facility/assets/src/views/ManageClassPage/CopyClassSidePanel.vue

```diff
@@ -163,8 +165,16 @@
       },
     },
     computed: {
-      isClassNameInvalid() {
-        return this.classroom.some(row => row[0] === this.copiedClassName) && !this.submitting;
+      classNameInvalidText() {
+        if (!this.submitting) {
+          const name = (this.copiedClassName || '').trim();
```

Comment:
non-blocking note - `trim()` won't be needed anymore when #13606 is merged

Reply:
Nah I think it's good to just merge - I think I may have only mentioned it because I came to this PR straight from the other one so trimming here was fresh on my mind :sweat_smile: